pravega/src/test/java/docs/javadsl/PravegaBaseTestCase.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.javadsl;
+
+import java.net.URI;
+
+import akka.stream.alpakka.pravega.ReaderSettingsBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import akka.actor.ActorSystem;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+
+import akka.testkit.javadsl.TestKit;
+
+import akka.stream.alpakka.pravega.ReaderSettings;
+
+import io.pravega.client.stream.impl.JavaSerializer;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+
+public abstract class PravegaBaseTestCase {
+  protected static final Logger LOGGER = LoggerFactory.getLogger(PravegaBaseTestCase.class);
+
+  protected static ActorSystem system;
+  protected static Materializer materializer;
+
+  // #configuration
+
+  protected ReaderSettings<String> settings =
+      ReaderSettingsBuilder.apply(system)
+          .withGroupName("javaGroupName")
+          .withReaderId("javaReaderId")
+          .withSerializer(new JavaSerializer<String>());
+
+  // #configuration
+  protected static String scope = "java-test-scope";
+
+  protected static String streamName = "java-test-topic";
+
+  @BeforeClass
+  public static void setup() {
+    system = ActorSystem.create();
+    materializer = ActorMaterializer.create(system);
+
+    StreamManager streamManager = StreamManager.create(URI.create("tcp://localhost:9090"));
+
+    if (streamManager.createScope(scope)) LOGGER.info("Created ", scope);
```

Comment:
```suggestion
    if (streamManager.createScope(scope)) LOGGER.info("Created scope [{}]", scope);
    else LOGGER.info("Reusing scope [{}]", scope);
```

I don't know what `false` means when invoking `streamManager.createScope` but it could be confusing if there'a trace (`"Created [my-scope]"`) that only appears sometimes.

Reply:
`streamManager.createScope` effectively returns (true) if the scope is newly created.
BTW your right, might be confusing.